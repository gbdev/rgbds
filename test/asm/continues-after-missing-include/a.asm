PUSHC
PUSHO
PUSHS
SECTION "test", WRAM0
UNION
INCLUDE "nonexistent1.inc"
WARN "still going!"
INCLUDE "nonexistent2.inc"
WARN "and going!"
ENDU
POPS
POPO
POPC
