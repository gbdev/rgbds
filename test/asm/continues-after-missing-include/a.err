warning: continues-after-missing-include/a.asm(7): [-Wuser]
    still going!
warning: continues-after-missing-include/a.asm(9): [-Wuser]
    and going!
