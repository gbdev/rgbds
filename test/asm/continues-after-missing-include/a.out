a.o: continues-after-missing-include/a.asm
a.o: nonexistent1.inc
a.o: nonexistent2.inc
